src/pages/ProductDetail/components/Detail/Navigate.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import './Navigate.scss';
+
+const Navigate = () => {
+  const navigate = useNavigate();
+
+  return (
+    <div className="navigate">
+      <form className="btns">
+        <button
+          className="gift btn"
+          onClick={() => {
+            navigate('/gift');
+          }}
+        >
+          선물
+        </button>
```

Comment:
조건없이 이동해야하는거라면 Link 컴포넌트를 사용할 수 있습니다.
```suggestion
        <Link className="gift btn" to="/gift">
          선물
        </Link>
```